### PR TITLE
Update rate review

### PR DESCRIPTION
**Pull request by hungsondo**
## Related Tickets:
- [#40704](https://edu-redmine.sun-asterisk.vn/issues/40704)

## WHAT this PR does?
- Update unlike review
- Update view app layout and book detail

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

File: app/Http/Controllers/ReviewController.php [hunk 1]
```diff
@@ -138,7 +138,13 @@ public function rate($id)
                 ->where('likeable_id', '=', $id)
                 ->get();
         if (count($like)) {
-            return redirect()->back()->with('error', __('messages.already-rate-this-review'));
+            Like::withTrashed()
+            ->where('user_id', getAuthUserId())
+            ->where('likeable_id', $id)
+            ->where('likeable_type', get_class($review))
+            ->forceDelete();
```
Comment (longnv-99) [inline app/Http/Controllers/ReviewController.php:145]: thụt zô

**Review by longnv-99 — APPROVED** [review]

**Review by truongcr7 — APPROVED** [review]

**Comment by hungsondo** [comment]
ready

File: resources/views/book-detail.blade.php [hunk 2]
```diff
@@ -178,7 +178,7 @@
 													{{ __('messages.edit') }}
 												</a>
 												<form action="{{ route('comments.destroy', $comment) }}" method="post"
-													onsubmit="return confirm(' {{ __('delete_confirm') }} ')">
+													onsubmit="return confirm(' {{ __('messages.delete_confirm') }} ')">
```
Comment (manhtd-1083) [inline resources/views/book-detail.blade.php:181]: bỏ onsubmit ra js riêng nhé

**Comment by hungsondo** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**